packages/beeq/src/components/alert/__tests__/bq-alert.e2e.ts

```diff
@@ -36,7 +36,7 @@ describe('bq-alert', () => {
 
     const element = await page.find('bq-alert');
     expect(element).toEqualAttribute('aria-hidden', 'true');
-    expect(element).toHaveClass('is-hidden');
+    expect(element).not.toHaveClass('is-hidden');
```

Comment:
I think this e2e test was good. But the new changes we were missing to set the `.is-hidden` class on componentDidLoad if `this.open = false`:

```ts
  componentDidLoad() {
    if (!this.open) {
      this.el.classList.add('is-hidden');
    }
  }
```